src/common/requests.js

```diff
@@ -0,0 +1,43 @@
+function request(method, url, headers, body) {
```

Comment:
Base part of the URL should be sourced from a constant. Example:

function request(method, path, headers, body)  {
   return fetch(`${API_BASE}/${path}`, {...})
}